var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -86,85 +100,53 @@ class Qt(Package):
-    depends_on("pcre2+multibyte", when='@5.9:')
-    depends_on("double-conversion", when='@5.7:')
-    depends_on("harfbuzz", when='@5:')
-
-    # Core options:
-    # -doubleconversion  [system/qt/no]
-    # -iconv             [posix/sun/gnu/no] (Unix only)
-    # -pcre              [system/qt]
-
-    # Gui, printing, widget options:
-    # -harfbuzz          [system/qt/no]
-    # -xkbcommon-x11     [system/qt/no]
-    # -system-xkbcommon
-
-    # Database options:
-    # -sqlite            [system/qt]
-
-    # Qt3D options:
-    # -assimp            [system/qt/no]
-
-    # QtQml
-    depends_on("python", when='@5.7.0:', type='build')
+    depends_on("inputproto", when='@:5.8')
+    depends_on("openssl@:1.0", when='@:5.9+ssl')
 
-    # OpenGL hardware acceleration
+    depends_on("glib", when='@4:')
+    depends_on("libpng", when='@4:')
+    depends_on("dbus", when='@4:+dbus')
     depends_on("gl@3.2:", when='@4:+opengl')
-    # xcb is Linux-specific
-    depends_on("libxcb", when=sys.platform != 'darwin')
-    depends_on("xcb-util-image", when=sys.platform != 'darwin')
-    depends_on("xcb-util-keysyms", when=sys.platform != 'darwin')
-    depends_on("xcb-util-wm", when=sys.platform != 'darwin')
-    depends_on("xcb-util-renderutil", when=sys.platform != 'darwin')
-    depends_on("libxkbcommon", when=sys.platform != 'darwin')
-    depends_on("inputproto", when='@:5.8')
-    depends_on("libx11", when=sys.platform != 'darwin')
 
+    depends_on("harfbuzz", when='@5:')
+    depends_on("double-conversion", when='@5.7:')
+    depends_on("pcre2+multibyte", when='@5.9:')
+
+    # Non-macOS dependencies and special macOS constraints
     if sys.platform != 'darwin':
+        depends_on("fontconfig")
+        depends_on("libx11")
+        depends_on("libxcb")
+        depends_on("libxkbcommon")
+        depends_on("xcb-util-image")
+        depends_on("xcb-util-keysyms")
+        depends_on("xcb-util-renderutil")
+        depends_on("xcb-util-wm")
         depends_on("libxext", when='@3:4.99')
-
-    # Webkit
-    depends_on("flex", when='+webkit', type='build')
-    depends_on("bison", when='+webkit', type='build')
-    depends_on("gperf", when='+webkit')
-
-    # Multimedia
-    # depends_on("gstreamer", when='+multimedia')
-    # depends_on("pulse", when='+multimedia')
-    # depends_on("flac", when='+multimedia')
-    # depends_on("ogg", when='+multimedia')
-    # -pulseaudio                [auto] (Unix only)
-    # -alsa                      [auto] (Unix only)
-
-    # Webengine options:
-    # -webengine-alsa            [auto] (Linux only)
-    # -webengine-pulseaudio      [auto] (Linux only)
-    # -webengine-embedded-build  [auto] (Linux only)
-    # -webengine-icu             [system/qt] (Linux only)
-    # -webengine-ffmpeg          [system/qt] (Linux only)
-    # -webengine-opus            [system/qt] (Linux only)
-    # -webengine-webp            [system/qt] (Linux only)
-
+    else:
+        for _spec in ('%gcc','%intel'):
```

Comment:
@sethrj there are a number of flake complaints, including no whitespace after the `,`

Reply:
Fixed. Sorry!